config/initializers/assets.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
```

Comment:
If I remember correctly, initializers are run in the order of filename. Do we need this initializer to run before our application specific ones since this touchers Rails config?

Reply:
Indeed they should run in the order of filename. Here I’m not sure it changes a lot of things though as usually initializers are used to configure various things in the app and the app really boots after the initializers.

Reply:
True, I think the risk is small here :+1: